### PR TITLE
[EMCAL-565]: Added ccdb entry framework.

### PR DESCRIPTION
**Pull request by hjbossi**

### CODE REVIEW

**Review by mfasDa — DISMISSED** [review]
Looks good, as we discussed.

File: Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALChannelCalibrator.h [hunk 1]
```diff
@@ -69,6 +70,9 @@ class EMCALChannelCalibrator : public o2::calibration::TimeSlotCalibration<o2::e
   void setIsTest(bool isTest) { mTest = isTest; }
   bool isTest() const { return mTest; }
 
+  CcdbObjectInfoVector getInfoVector() { return mInfoVector; }
+  std::vector<DataOutput> getOutputVector() { return mCalibObjectVector; }
```
Comment (shahor02) [inline Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALChannelCalibrator.h]: do you really want to create a copy of returned objects? These methods should be declared as `const`

Comment (hjbossi) [reply Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALChannelCalibrator.h]: Hi @shahor02! This is now fixed.

**Comment by mfasDa** [comment]
CI errors seem to be unrelated

**Review by shahor02 — APPROVED** [review]
CI failures unrelated, merging

**Pull request merged**